### PR TITLE
added balloon notifications to DB and File imports

### PR DESCRIPTION
**Pull request by Atlas139mkm**
### Description of the Change

I have removed pop up notification on file import and replaced it with a balloon notification. I have also added the notification to database imports as well.

File Importer:
image
This displays the exact same message as before but in balloon format.

Database:
image
It says : imported 18393 rows of data from connection: test (what I named my connection)

### Alternate Designs

I have not added a click for more details. I wanted to wait for feedback first and what ask details I should add to that if I did add that feature. Also how to add a link to a balloon notification.

### Why Should This Be In Core?

It achieves the same outcome as the current pop up but does it in a less intrusive way to the user.

It also adds the functionality to the DB connection importer.

### Benefits

UX improvements.

### Possible Drawbacks

Maybe less obvious for some people??

### Verification Process

I ensured the message still posted the appropriate messages that I did before. This was based on if a file was imported, if a file wasn't imported and If no file was present.

### Applicable Issues

#1309

### CODE REVIEW

**Review by arcturus2 — APPROVED** [review]
Thanks for this @Atlas139mkm. This works really well. A very minor fix if you can make it is when it is only 1 file then it should be singular instead of "files".

**Review by antares1470 — CHANGES_REQUESTED** [review]
Agreeing with the suggestion from @arcturus2. Looks good otherwise

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java [hunk 1]
```diff
@@ -138,33 +138,39 @@ public PluginParameters createParameters() {
...
     private void displaySummaryAlert(final int importedRows, final List<String> validFilenames, final List<String> invalidFilenames) {
         Platform.runLater(() -> {
-            boolean success = true;
             final StringBuilder sbHeader = new StringBuilder();
             final StringBuilder sbMessage = new StringBuilder();
 
+            String fileFiles = (validFilenames.size() > 1) ? "files" : "file";
+            if (validFilenames.size() == 0) {
+                fileFiles = "files";
+            }
```
Comment (antares1470) [inline CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java:147]: Can be simplified to `final String fileFiles = (validFilenames.size() == 1) ? "file" : "files";`

File: CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java [hunk 2]
```diff
@@ -138,33 +138,39 @@ public PluginParameters createParameters() {
...
+                sbHeader.append(String.format("Imported %d rows of data from %d " + fileFiles, importedRows, validFilenames.size()));
+                sbMessage.append("The following " + fileFiles + " contained data:");
                 validFilenames.forEach(filename -> {
                     sbMessage.append("\n  ");
                     sbMessage.append(filename);
                 });
+                String invalidFileFiles = (invalidFilenames.size() > 1) ? "files" : "file";
+                if (invalidFilenames.size() == 0) {
+                    invalidFileFiles = "files";
+                }
```
Comment (antares1470) [inline CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java:161]: similar here

**Review by antares1470 — CHANGES_REQUESTED** [review]

**Review by antares1470 — APPROVED** [review]

**Pull request merged**